deployment/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+services:
+  mosquitto:
+    image: eclipse-mosquitto
+    container_name: mosquitto
+    ports:
+      - 1883:1883
+    networks:
+      - mqtt2influxdb-network
+    volumes:
+      - "./mosquitto.conf:/mosquitto/config/mosquitto.conf"
+      - "./users.conf:/mosquitto/config/users.conf"
+      - "./mosquitto/data:/mosquitto/data"
+      - "./mosquitto/log:/mosquitto/log"
+    restart: unless-stopped
+  mqtt2influxdb:
```

Comment:
i would split the services by an empty line, but it depens on you if you want to do it...